Services/ManagerServiceTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using AutoMapper;
+using Managers.BLL.Interfaces;
+using Managers.BLL.Services;
+using Managers.DAL.Entities;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace BLL.UnitTests.Services
+{
+    public class ManagerServiceTests
+    {
+        [Theory]
+        [InlineData(3)]
+        public void GetById_ShouldReturnManager(int id)
+        {
+            //arrange
+            var manager = new Manager { Id = 3, FirstName = "Vladimir" };

```

Comment:
Use the id argument obtained from InlineData for instantiating a Manager object. Move the "Vladimir" literal to InlineData. It will be possible to add multiple sets of InlineData